engine/core/IgeEntity.js

```diff
@@ -847,6 +847,7 @@ var IgeEntity = IgeObject.extend({
 
 			if (ige.isClient) {
 				ige.client.emit('width', {entity: this, px: px});
+				this.emit('width', this._bounds2d.x);
```

Comment:
I think `width` and `height` events can be replaced with single one `size` like this:
```javascript
this.emit('size', [{
	w: this._bounds2d.x,
	h: this._bounds2d.y
}]);
```

and it requires only one unified event listener on rendering side